tests/integration/test_cmorize_obs.py

```diff
@@ -5,6 +5,7 @@
 import sys
 
 import iris
+from iris import coord_systems, fileformats
```

Comment:
we should not do this just here but rather have a general fix like here https://github.com/ESMValGroup/ESMValCore/pull/1367

Reply:
That PR certainly is more comprehensive. Still, the imports here look like perfectly idiomatic Python to me, so I see no reason to discourage them.